Update sharepoint-pnp-cmdlets.md (#173)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/docs-conceptual/sharepoint-pnp/sharepoint-pnp-cmdlets.md

# PnP PowerShell overview

> PnP PowerShell is a .NET Core 3.1 / .NET Framework 4.6.1 based PowerShell Module providing over 600 cmdlets that work with Microsoft 365 environments and more specifically SharePoint Online and Microsoft Teams.

Reviewer: ```suggestion
PnP PowerShell is a .NET Core 3.1/.NET Framework 4.6.1 based PowerShell Module providing over 600 cmdlets that work with Microsoft 365 environments and more specifically SharePoint Online and Microsoft Teams.
```

> For more information about installing or upgrading to this module, please refer to the documentation at https://pnp.github.io/powershell/articles/index.html

Reviewer: ```suggestion
For more information about installing or upgrading to this module, refer to these [PnP articles](https://pnp.github.io/powershell/articles/index.html).
```

## Supportability and SLA for PnP PowerShell cmdlets

> This library is open-source and community provided library with active community providing support for it. This is not Microsoft provided module so there's no SLA or direct support for this open-source component from Microsoft. Please report any issues using the [issues list in GitHub](https://github.com/pnp/powershell/issues).

Reviewer: ```suggestion
This library is an open-source and community-provided library with an active community providing support for it. It is not a Microsoft-provided module so there's no SLA or direct support for this open-source component from Microsoft. You can report any issues using the [issues list in GitHub](https://github.com/pnp/powershell/issues).
```
